speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/MediatorTests.cpp

```diff
@@ -3,21 +3,32 @@
 #include "ConfigEnums.h"
 #include "Mediator.h"
 #include "Mediator.inl"
+#include "SDAConfig.h"
 #include "mocks/DecisionMakerMock.h"
 #include <sys/types.h>
 #include <sys/stat.h>
 #define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING 1
 #include <experimental/filesystem>
 #include "../rppUtils/RppUtils.hpp"
 
-#define MockMediator Mediator<DecisionMakerMock>
+/// @brief A mediator that uses the standard SDecisionMakerMock
+#define MockMediator Mediator<SDecisionMakerMock>
+
+/// @brief A mediator that uses SDAConfig in DecisionmakerMock internally
+#define SDAConfigMediator Mediator<DecisionMakerMock<SDAConfig>>
+
+#define TEST_AMOUNT 20
```

Comment:
Add a comment explaining the defintion of this define, e.g. "The number of trials to run for nondeterministic tests"